src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -212,18 +212,17 @@ export const Map: React.FC<MapProps> = ({
     setSelectedUserProfile,
   });
 
-  const roomOverlay = useMemo(
-    () =>
-      venue.rooms?.map((room) => (
-        <MapRoom
-          key={room.title}
-          venue={venue}
-          room={room}
-          selectRoom={() => selectRoom(room)}
-        />
-      )),
-    [selectRoom, venue]
-  );
+  const roomOverlay = useMemo(() => {
+    const enabledRooms = venue?.rooms?.filter((room) => room.isEnabled) ?? [];
+    return enabledRooms.map((room) => (
+      <MapRoom
+        key={room.title}
+        venue={venue}
+        room={room}
```

Comment:
```
return venue?.rooms?
.filter(...)
.map(...)
```

Reply:
Can we avoid chaining please? This won't save any lines of code or anything. Unless you are both up for this I'd rather stay with the named constant.

Reply:
[nit], you can merge it now, but this way it looks cleaner and you will actually save a few lines of code by removing `return` + will reduce the cognitive load + we will not create an unnecessary `const`

Reply:
Okey, I've changed my mind. You can take a look.

Reply:
Thanks

Reply:
[/2c] I'm a fan of extracting `const`s when it will make the statement simpler to read/cognitively process/etc, but in this case, agree that it is probably 'cleaner' chained.